templates/base.html

```diff
@@ -86,6 +86,17 @@
                 <span class="icon-bar"></span>
             </button>
 
+            <a id="base-url" class="navbar-brand" href="{% url 'hc-index' %}">
```

Comment:
With this change, the logo is shown unconditionally, also in the list of checks page, project's settings page, check's details page etc.

I prefer to leave things as is: have branding on the "public" pages, and on the project overview, but not elsewhere. It adds visual clutter and serves no purpose.